spec/services/idv/steps/ipp/ssn_step_spec.rb

```diff
@@ -1,12 +1,14 @@
 require 'rails_helper'
 
 describe Idv::Steps::Ipp::SsnStep do
+  let(:params) do { doc_auth: {} } end
```

Comment:
For consistency, I prefer to use `do ... end` only for multiline blocks, so this line just gets super squiggly

```suggestion
  let(:params) { { doc_auth: {} } }
```

Reply:
Done in e07b98fe2